gent_base.theme

```diff
@@ -60,6 +71,20 @@ function _gent_base_webform_pre_render(array $form) {
     _gent_base_loop_webform_elements($form['elements']);
   }
 
+  $form['#webform'] = $webform = Webform::load($form['#webform_id']);
+
+  // Check if the form has input fields.
+  if ($webform->hasWizardPages()) {
+    $currentPage = $form['progress']['#current_page'];
```

Comment:
Lokale vars in een functie mogen altijd underscored zijn.

Reply:
Done 😉